wordpress-mobile/WordPress-iOS
# Login: Social Error Screens

Refs #7675

Error display screen for when login to Google succeeds, but login using the token with WordPress.com subsequently fails.

![simulator screen shot - iphone 8 - 2017-10-01 at 23 12 13](https://user-images.githubusercontent.com/517257/31064912-30cf814a-a6fe-11e7-9715-3b07618c4c03.png)

To test:
- login to a Google account with an email that doesn't have a wpcom account
- ensure error screen matches mockup

### Note:
This is forked from #7872 and can't be merged until that PR is approved & merged.

Needs review: @aerych
